module/Motion/CorrectingRotation.cpp

```diff
@@ -0,0 +1,171 @@
+/**
+ * @file   CorrectingRotation.h
+ * @brief  角度補正回頭
+ * @author aridome222
+ */
+
+#include "CorrectingRotation.h"
+
+using namespace std;
+
+CorrectingRotation::CorrectingRotation(int _targetAngle, double _targetSpeed)
+  : targetAngle(_targetAngle), targetSpeed(_targetSpeed){};
+
+void CorrectingRotation::run()
+{
+  // 事前条件を判定する
+  if(!isMetPrecondition()) {
+    return;
+  }
+
+  /*
+   * rear_camera.shより直線の角度を取得
+   * シミュレータ環境ではpopenが使えないため，
+   * ファイルに出力して読み込む
+   */
+  FILE* fp = fopen("result.txt", "r");
```

Comment:
下の分岐でopenするならここではopenしない方がいいと思う
```suggestion
  FILE* fp = nullptr;
```